package.json

```diff
@@ -104,7 +105,9 @@
     "karma-phantomjs-launcher": "^1.0.1",
     "merge-stream": "^1.0.0",
     "mocha": "^2.3.3",
+    "myanmar-tools": "^1.1.3",
```

Comment:
Why is `myanmar-tools` listed in both the `dependencies` and `devDependencies`? If it is included in the `dependencies`, then it will install for both development _and_ production

Reply:
Oops, thanks for catching that! Will update.